### ISSUE TITLE
allowJs not respected so .js and .d.ts files not copied over to dist

### ISSUE THREAD

**Issue by thomasballinger** [isaacs/tshy#90]
I have some .js and .d.ts files in src/ I'd like to end up in dist/esm/ but tshy seems to ignore `"allowJs": true` which makes this work with straight `tsc`.

Is this something that makes sense to support?

repro: `git clone https://github.com/thomasballinger/tshy-repro; npm i; npx tshy` and note that src/raw.js doesn't end up in dist/esm.

**Comment by shaungrady** [comment]
Would love to see this as well. Just hit this limitation.

**Comment by shellscape** [comment]
hit this as well this evening.

**Comment by isaacs** [comment]
No, the way to do this is to put them somewhere else, like `lib/`. Tshy is opinionated about `src/` being the place for TS.
